src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -2,18 +2,47 @@
 
 import core.basesyntax.Storage;
 
+@SuppressWarnings("unchecked")
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int Capacity = 10;
```

Comment:
```suggestion
    private static final int CAPACITY = 10;
```
Use the correct name style